src/types/configYml.ts

```diff
@@ -3,4 +3,5 @@ import { ProdPermalink, StagingPermalink } from "./pages"
 export type ConfigYmlData = {
   staging?: StagingPermalink
   prod?: ProdPermalink
+  "facebook-pixel": number | null
```

Comment:
why's the type altered here?

Reply:
Hmmm rather than allowing for `null`, could we just make this key optional instead?

Reply:
This is just for the purposes of testing! `ConfigYmlData` doesn't have the full typing of data available